pkg/apis/serving/helpers/contexts.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import "context"
+
+type (
+	revCtxKey struct{}
+)
+
+type revCtx struct {
+	anns map[string]string
```

Comment:
would prefer `annotations` (or `annos`)  but as its a private I leave it up to you

Reply:
I removed that part as suggested by @julz .